kafka-client/src/main/java/dev/responsive/kafka/store/AtomicWriter.java

```diff
@@ -108,9 +113,11 @@ public int partition() {
   }
 
   public AtomicWriteResult setOffset(final long offset) {
-    final var result = client.execute(
-        plugin.metadata().setOffset(tableName, partition, offset, epoch));
+    final BatchStatementBuilder builder = new BatchStatementBuilder(BatchType.UNLOGGED);
```

Comment:
we now use a batch instead of a single statement so that a set offset statement isn't assumed to be a LWT